pyproject.toml

```diff
@@ -0,0 +1,76 @@
+[project]
+name = "molstar-volseg"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.10"
+dependencies = [
+  "asgiref==3.5.0",
```

Comment:
Historically, we had issues with the exact pins. Might have to loosen them.

Is there some way to have a single source of truth for both this and the conda environments?